src/_cffi_src/openssl/objects.py

```diff
@@ -9,6 +9,14 @@
 """
 
 TYPES = """
+typedef struct {
+    int type;
+    int alias;
+    const char *name;
+    const char *data;
+} OBJ_NAME;
```

Comment:
The struct is still available in master: https://github.com/openssl/openssl/blob/master/include/openssl/objects.h#L935